### DISCUSSION TITLE
能否支持忽略词列表？

### DISCUSSION THREAD

**Discussion by sangshuduo** [shibing624/pycorrector#435] [Ideas]
某些领域误报率比较高，又没有足够多的数据来训练，如果可以支持指定一个忽略词列表会方便很多

**Comment by shibing624** [comment]
这个功能：https://github.com/shibing624/pycorrector#%E8%87%AA%E5%AE%9A%E4%B9%89%E6%B7%B7%E6%B7%86%E9%9B%86 满足你要求

**Reply by sangshuduo** [reply]
谢谢，能否提供命令行参数传入的使用方法？

**Reply by shibing624** [reply]
没有计划加入到命令行中。

**Reply by sangshuduo** [reply]
OK, thanks.
